templates/scripts/setup-user-password.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
How about calling this "create-temporary-passwords.sh" or something?

Reply:
yes, use this "create-temporary-passwords.sh"